dali/base/dasds.cpp

```diff
@@ -8116,8 +8139,9 @@ void CCovenSDSManager::handleNodeNotify(notifications n, CServerRemoteTree &tree
     }
 }
 
-void CCovenSDSManager::handleNotify(CSubscriberContainerBase &subscriber, MemoryBuffer &notifyData)
```

Comment:
It could have been, when something passes ownership like this, I prefer (for no good reason maybe) a trailing pointer versus reference